Geometry/RPCGeometryBuilder/src/RPCGeometryBuilderFromDDD.h

```diff
@@ -33,7 +33,7 @@ class RPCGeometryBuilderFromDDD
 
  private:
   RPCGeometry* buildGeometry(DDFilteredView& fview, const MuonDDDConstants& muonConstants);
-  std::map<RPCDetId,std::list<RPCRoll *> > chids;
+  std::map<RPCDetId,std::set<RPCRoll *> > chids;
```

Comment:
this randomizes the order of RPCRoll between jobs or IOVs.
Not sure if this is fully harmless.